harmonica/_transformations.py

```diff
@@ -393,6 +393,68 @@ def total_gradient_amplitude(grid):
     return np.sqrt(gradient[0] ** 2 + gradient[1] ** 2 + gradient[2] ** 2)
 
 
+def tilt_angle(grid):
+    r"""
+    Calculates the tilt angle of a potential field grid
+
+    Compute the tilt of a regular gridded potential field
+    :math:`M`. The horizontal derivatives are calculated
+    though finite-differences while the upward derivative
+    is calculated using FFT.
+
+    Parameters
+    ----------
+    grid : :class:`xarray.DataArray`
+        A two dimensional :class:`xarray.DataArray` whose coordinates are
+        evenly spaced (regular grid). Its dimensions should be in the following
+        order: *northing*, *easting*. Its coordinates should be defined in the
+        same units.
+
+    Returns
+    -------
+    tilt_grid : :class:`xarray.DataArray`
+        A :class:`xarray.DataArray` after calculating the
+        tilt of the passed ``grid``.
```

Comment:
Just adding the units of the returned DataArray
```suggestion
        A :class:`xarray.DataArray` with the calculated tilt
        in radians.
```